includes/admin/meta-boxes/views/html-related-orders-row.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Display a row in the related orders table for a subscription or order
  *
- * @var array $order A WC_Order or WC_Subscription order object to display
+ * @var WC_Order|WC_Subscription $row_order A WC_Order or WC_Subscription order object to display.
```

Comment:
I changed this mostly to address my readability concerns, i.e. is this the "global" order for the page or the order for a single row?

But since this will impact backwards compat, I'll revert this to `$order`. Thanks 🙌

Reply:
Resolved in d423542.

Do you think it is safe to keep the `WC_Order|WC_Subscription` type change in the docblock?